### DISCUSSION TITLE
大佬，我登上去了，请教下能帮我的邮箱建个账号吗？

### DISCUSSION THREAD

**Discussion by vasen-plugin** [MagicPigZHN/Exhentai-Shared-Account#4] [Q&A] [answered]
我用的是vpn，而且有表站账号，但是一直都转不到里站，请问你是用什么方法注册的，如果可以，能帮我的google邮箱建个账号吗

**Comment by Mpigest** [comment, marked as answer]
> 我用的是vpn，而且有表站账号，但是一直都转不到里站，请问你是用什么方法注册的，如果可以，能帮我的google邮箱建个账号吗

1.  我在 [**discussion#1**](https://github.com/MagicPigZHN/Exhentai-Shared-Account/discussions/1) 中有讲述如何成功注册一个里站账号

2. 本人不提供代开账号的服务，还请自己注册，注册一个谷歌账号并不难
